lib/bolt/bolt_option_parser.rb

```diff
@@ -761,6 +761,15 @@ def initialize(options)
         @options[:'save-rerun'] = save
       end
 
+      separator "\nREMOTE ENVIRONMENT OPTIONS"
+      define('--env-var ENVIRONMENT_VARIABLES', 'Environment variables to set on the target') do |envvar|
+        unless envvar.include?('=')
+          raise Bolt::CLIError, "Environment variables must be specified using 'myenvvar=key' format"
+        end
+        @options[:env_vars] ||= {}
+        @options[:env_vars].merge!(Hash[*envvar.split('=', 2)])
```

Comment:
I think `@options[:env_vars].store(*envvar.split('=', 2))` would work here and avoid the conversion to `Hash`.